test/utils/dom-parser.spec.js

```diff
@@ -0,0 +1,18 @@
+import domParser from '../../src/js/utils/dom-parser'
+
+describe.only('dom parser spec', () => {
+  let elementString
+
+  beforeEach(() => {
+    elementString = '<span>sample text</span>'
+  })
+
+  it('should create a dom element', () => {
+    const tempElement = domParser(elementString)
+    expect(tempElement.firstChild).to.be.an.instanceof(Element)
+  })
+
+  it('should not create a dom element', () => {
```

Comment:
Pra ser mais específico com o que de fato esse exemplo testa a descrição poderia ser `should throw an error`, o que acha?

Reply:
Sim, faz sentido. Alterando ;)